src/main/java/folk/sisby/surveyor/terrain/ChunkSummary.java

```diff
@@ -76,7 +76,7 @@ public ChunkSummary(World world, WorldChunk chunk, int[] layerHeights, RegistryP
 						} else { // Blocks Movement or Has Non-Water Fluid.
 							if (foundFloor == null) {
 								if (carpetPos.getY() == y + 1) {
-									foundFloor = new LayerSummary.FloorSummary(carpetPos.getY(), biomePalette.findOrAdd(section.getBiomeEntry(x, carpetPos.getY(), z, world.getBottomY(), world.getTopY()).value()), blockPalette.findOrAdd(carpetBlock), world.getLightLevel(LightType.BLOCK, carpetPos), waterDepth, waterDepth == 0 ? 0 : world.getLightLevel(LightType.BLOCK, pos.up().up(waterDepth)));
```

Comment:
for completeness, it's always good to confirm a surveyor update with a hoofprint update as well, just because atlas hides a lot of detail that could become wrong when touching the scanner. I think you're fine here because these are just renames, i can only assume the impl is the same.

Reply:
i just got an initial port going but i'm encountering the same rendering issues as in aa4. though considering the fact that hoofprint has way less parts i think i could reasonably try to figure it out